cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -175,6 +177,92 @@ public String cruiseControlHost(String clusterName, String clusterNamespace) {
                 });
     }
 
+    @Test
```

Comment:
Just a nit but these tests should be placed at the bottom with the other tests below the utility methods.